appveyor.yml

```diff
@@ -59,6 +60,7 @@ build_script:
       -DMRN_GROONGA_NORMALIZER_MYSQL_EMBED=OFF
       -DGRN_WITH_BUNDLED_LZ4=ON
       -DGRN_WITH_BUNDLED_MECAB=ON
+      -DGRN_WITH_BUNDLED_MESSAGE_PACK=ON
```

Comment:
Do we need to specify this?
Does https://github.com/mroonga/mroonga/pull/232/files#diff-af3b638bc2a3e6c650974192a53c7291R131 enable this automatically?

Reply:
When building the Mroonga in the AppVeyor, that option was ignored...
the Mroonga had not the Message pack as below.

```
C:\Users\Mroonga\Downloads\mariadb-10.1.38-with-mroonga-9.00-winx64 (1)\mariadb-
10.1.38-winx64\bin>groonga.exe --version
Groonga 9.0.0 [Windows,AMD64,utf8,match-escalation-threshold=0,nfkc,mecab,onigmo
,lz4]

configure options: <>
```

Therefore, this option is needed for bundling the Message pack in the Mroonga that built by the AppVeyor.